frontend/src/components/Question.jsx

```diff
@@ -0,0 +1,138 @@
+import { useState, useEffect } from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { Card, CardContent, IconButton, Stack } from '@mui/material'
+import ArrowCircleRightIcon from '@mui/icons-material/ArrowCircleRight'
+import ArrowCircleLeftIcon from '@mui/icons-material/ArrowCircleLeft'
+import QuestionButton from './QuestionButton'
+import styled from '@emotion/styled'
+import { Typography } from '@mui/material'
+
+export const Heading = styled(Typography)`
+    font-size: 50px;
+`
+
+export const QuestionOptionsContainer = styled.div`
+    margin: 50px;
+`
+
+export const QuestionPageContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+    margin-top: 100px;
+`
+
+function Question() {
+    const { id } = useParams()
+    const questionId = Number(id)
+    const navigate = useNavigate()
+
+    const [selectedOption, setSelectedOption] = useState(null)
+    const [questionData, setQuestionData] = useState(null)
+    const [totalQuestions, setTotalQuestions] = useState(null)
+
+    useEffect(() => {
+        const fetchTotalQuestions = async () => {
+            try {
+                const response = await fetch(
+                    `http://127.0.0.1:5000/api/question`
```

Comment:
Vite will proxy the request to the backend and in staging/production the request will be handled by the Route object and forwarded to the backend.

```suggestion
                    `/api/question`
```